src/main/java/ch/njol/skript/expressions/ExprAttacked.java

```diff
@@ -50,13 +52,18 @@
  * @author Peter Güttinger
  */
 @Name("Attacked")
-@Description("The victim of a damage event, e.g. when a player attacks a zombie this expression represents the zombie.")
+@Description("The victim of a damage event, e.g. when a player attacks a zombie this expression represents the zombie. " +
+	"This also covers the hit entity in a projectile hit event.")
 @Examples({"on damage:",
 		"	victim is a creeper",
 		"	damage the attacked by 1 heart"})
-@Since("1.3")
-@Events({"damage", "death"})
+@Since("1.3, INSERT VERSION (projectile hit event)")
+@Events({"damage", "death", "projectile hit"})
+@RequiredPlugins("Minecraft 1.11+")
```

Comment:
The @RequiredPlugins tag might be misleading on this expression considering it works on older versions, just not in the projectile hit event. You're probably better off just removing it and specifying the version requirement in the description.